ci/helm/preview/values.yaml

```diff
@@ -0,0 +1,93 @@
+expose:
+  Annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: hook-succeeded
+  config:
+    # add tls configuration
+    domain: platform.dev.nuxeo.com
+    exposer: Ingress
+    http: true
+    tlsacme: true
+    tlsSecretName: platform-cluster-tls
+    urltemplate: '"{{.Service}}-{{.Namespace}}.{{.Domain}}"'
+
+cleanup:
+  Args:
+    - --cleanup
+  Annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: hook-succeeded
+
+nuxeo:
+  fullnameOverride: preview
+  nuxeo:
+    customParams: |-
+      elasticsearch.restClient.socketTimeoutMs=300000
+      elasticsearch.restClient.connectionTimeoutMs=300000
+    podAnnotations:
+      preview/scm.ref: ${SCM_REF}
+    podLabels:
+      branch: "$BRANCH_NAME"
+      team: platform
+      resource: pod
+      usage: preview
+    service:
+      annotations:
+        fabric8.io/ingress.annotations: |-
+          nginx.ingress.kubernetes.io/proxy-body-size: 0
+          nginx.ingress.kubernetes.io/server-snippet: ignore_invalid_headers off;
+      labels:
+        team: platform
+        resource: service
+        usage: preview
+    image:
+      repository: ${DOCKER_REGISTRY}/nuxeo/nuxeo
+      tag: ${VERSION}
+      pullPolicy: Always
+    packages: nuxeo-web-ui nuxeo-dam
+    livenessProbe:
+      initialDelaySeconds: 900
+      periodSeconds: 20
+      successThreshold: 1
+      failureThreshold: 5
+      timeoutSeconds: 10
+    readinessProbe:
+      periodSeconds: 20
+      initialDelaySeconds: 600
+      successThreshold: 1
+      timeoutSeconds: 10
+      failureThreshold: 5
+    custom_envs:
+    - name: NUXEO_CLID
+      valueFrom:
+        secretKeyRef:
+          name: instance-clid
+          key: instance.clid
+    persistence:
+      enabled: ${PERSISTENCE}
+  mongodb:
+    deploy: ${PERSISTENCE}
+    persistence:
+      enabled: ${PERSISTENCE}
+    labels:
+      branch: "$BRANCH_NAME"
+      resource: deployment
+      team: platform
+      usage: preview
+    podLabels:
+      branch: "$BRANCH_NAME"
+      resource: pod
+      team: platform
+      usage: preview
+  elasticsearch:
+    deploy: ${PERSISTENCE}
+    master:
+      persistence:
+        enabled: ${PERSISTENCE}
+    data:
+      persistence:
+        enabled: ${PERSISTENCE}
+    cluster:
+      env:
+        EXPECTED_MASTER_NODES: 1
+        RECOVER_AFTER_MASTER_NODES: 1
```

Comment:
This should be in [nuxeo-helm-chart](https://github.com/nuxeo/nuxeo-helm-chart) repository.

Reply:
Yes probably, it depends on if we want them all the time or just for the preview.
But in fact:
- What does these values correspond to? I cannot find the in the elasticsearch Helm chart: https://github.com/helm/charts/blob/master/stable/elasticsearch/values.yaml.
- Why do we need them set this way?

Reply:
In the context of this PR preview environment, we need them for the cluster to recover because it didn't find preview-elasticsearch-discovery service. The non-existence of the service is strange, it should exist at the same time than pod is created.

These environment values are used in the config map of elasticsearch chart: https://github.com/helm/charts/blob/master/stable/elasticsearch/templates/configmap.yaml
We can see that `MINIMUM_MASTER_NODES`, `EXPECTED_MASTER_NODES ` and `RECOVER_AFTER_MASTER_NODES ` all default to `2` whereas we set it to `1` in nuxeo helm chart: https://github.com/nuxeo/nuxeo-helm-chart/blob/master/nuxeo/values.yaml#L110
I think it makes sense to also declare these two variables as we're lowering the number of required master nodes, what do you think?

Reply:
Great! Yes, let's add them tot he nuxeo Helm chart.
NXP-28618 can be used for the commit message.

Reply:
Do I keep this patch in the preview chart then ? 
Wouldn't hurt to have it while waiting for the newer version of nuxeo helm chart.

Reply:
You can do a PR on the nuxeo chart and reference this PR version in the requirements.
Then when the chart is merged `version: ~1.0.0` should fetch the latest release.

Reply:
For you information, as I said, instead of keeping the Helm chart changes here, you could have referenced the PR version of the nuxeo Helm chart built here: https://jenkins.platform.dev.nuxeo.com/job/nuxeo/job/nuxeo-helm-chart/view/change-requests/job/PR-8/3/console (look for `nuxeo-1.0.5-PR-8-3.tgz`).

In [requirements.yaml](https://github.com/nuxeo/nuxeo/blob/feature-NXP-27713-nuxeo-staging-preview/ci/helm/preview/requirements.yaml#L12), just use:
```
dependencies:
  - alias: nuxeo
    name: nuxeo
    version: 1.0.5-PR-8-3
```

```
```

Reply:
Done here https://jenkins.platform.dev.nuxeo.com/blue/organizations/jenkins/nuxeo%2Fnuxeo/detail/PR-3856/59/pipeline

https://jenkins.platform.dev.nuxeo.com/job/nuxeo/job/nuxeo/job/PR-3856/59/artifact/ci/helm/preview/requirements.lock